nucypher/policy/conditions/lingo.py

```diff
@@ -330,6 +330,18 @@ def __init__(self, operand: Condition):
     "float": lambda a: float(a),
     "int": lambda a: int(a),
     "str": lambda a: str(a),
+    # JSON conversion
+    "fromJson": lambda a: json.loads(a if isinstance(a, str) else a.decode()),
```

Comment:
`json.loads` already takes care of typing for us:
```
def loads(s, *, cls=None, object_hook=None, parse_float=None,
        parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
    """Deserialize ``s`` (a ``str``, ``bytes`` or ``bytearray`` instance
    containing a JSON document) to a Python object.
```

```suggestion
    "fromJson": lambda a: json.loads(a),
```